ztag/schema.py

```diff
@@ -1245,6 +1271,11 @@ class LintBool(String):
                     "banner": ztag_mysql,
                 }, category="3306/MySQL"),
             }),
+            Port(27017): SubRecord({
```

Comment:
**Question** ([_pull request commenting with a purpose_](https://gist.github.com/andrewsardone/3fb9746e415d59784e97))

@parkanzky: perhaps this is more of a relevant question to https://github.com/zmap/zgrab2/pull/156, but should we be scanning [ports `27018` & `27019`](https://docs.mongodb.com/manual/reference/default-mongodb-port/)?

/cc @justinbastress, @dadrian

Reply:
Let's start out with whichever one the `mongo` client connects to by default, and add the other ones later.